src/Parser/NoBody.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace React\Http\Parser;
+
+use Evenement\EventEmitterTrait;
+use React\Http\Request;
+
+class NoBody implements ParserInterface
```

Comment:
This concept looks a bit unclear to me, what is a `NoBody` parser supposed to do?

Looks like taking SRP one step too far… :)